docs/cni/rkt/README.md

```diff
@@ -140,8 +137,7 @@ With the networks created, let's start a container on `calico-01`
 ### On calico-01
 Create a webserver using an `nginx` container. We'll put this on `net1` network.
 
-Because `rkt` doesn't support daemonizing containers directly, we'll use [systemd-run][systemd-run]
-to create a transient systemd service.
+We'll use systemd run to create the webserver service.
 
     sudo systemd-run rkt run --store-only=true --insecure-options=image --net=net1 --mount volume=myvol,target=/var/run --volume=myvol,kind=empty docker://nginx
  
```

Comment:
Here was my output for the record, in case you want to edit it in:

```
core@calico-01 ~ $ sudo rkt list
UUID            APP     IMAGE NAME                                      STATE  NETWORKS
65f026a5        node    registry-1.docker.io/calico/node:latest         running
b2dd9cff        nginx   registry-1.docker.io/library/nginx:latest       runningnet1:ip4=192.168.0.0, default-restricted:ip4=172.16.28.2
```